tools/update_runnablejobs.py

```diff
@@ -193,16 +171,20 @@ def add_jobs_to_jobpriority(new_data=None, priority=1, timeout=0, set_expired=Fa
 
     # Now loop through the reduced jobs and update the database as needed
     for job in new_data['results']:
+        # XXX: from earlier in the code, I believe we could have two jobs one with the normal
+        # buildsystem type and the other with *
         _buildsystem = job["build_system_type"]
```

Comment:
we cannot have duplicates in our jobpriorities table- we either need no * and to support explicitly 'buildbot' and 'taskcluster' with about 800 duplicated jobs.  That makes it very difficult to support preseed.json, but it could be done.  The only case I see for not doing '*' is if we expect different failure patterns on taskcluster vs buildbot when we run side by side.  By failures, I am not referring to intermittent, but real failures that cause a backout or hotfix.